WMFModel/WMFArticle+CoreDataProperties.h

```diff
@@ -0,0 +1,29 @@
+#import "WMFArticle+CoreDataClass.h"
+
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface WMFArticle (CoreDataProperties)
+
++ (NSFetchRequest<WMFArticle *> *)fetchRequest;
+
+@property (nonatomic) BOOL isBlocked;
+@property (nullable, nonatomic, copy) NSString *key;
+@property (nullable, nonatomic, copy) NSDate *viewedDate;
+@property (nullable, nonatomic, copy) NSDate *viewedCalendarDate;
+@property (nullable, nonatomic, copy) NSString *viewedFragment;
+@property (nonatomic) double viewedScrollPosition;
+@property (nullable, nonatomic, copy) NSDate *newsNotificationDate;
+@property (nullable, nonatomic, copy) NSDate *savedDate;
+@property (nonatomic) BOOL wasSignificantlyViewed;
+@property (nullable, nonatomic, copy) NSString *displayTitle;
+@property (nullable, nonatomic, copy) NSString *wikidataDescription;
+@property (nullable, nonatomic, copy) NSString *snippet;
+@property (nullable, nonatomic, copy) NSString *thumbnailURLString;
+@property (nonatomic) double latitude;
```

Comment:
This could also be a CLLocation directly via a transformable property. We probably wouldn't be able to reliably query on it though.